schema.hh

```diff
@@ -635,6 +636,7 @@ private:
         // The flag is not stored in the schema mutation and does not affects schema digest.
         // It is set locally on a system tables that should be extra durable
         bool _wait_for_sync = false; // true if all writes using this schema have to be synced immediately by commitlog
+        std::shared_ptr<dht::i_partitioner> _partitioner;
```

Comment:
The comment above shoudl be extended to _partitioner. Digests and what they include or not are a major bug source, so we should be careful.

Reply:
done.